### PR TITLE
Fixes ear+eye insta kills

### PR DESCRIPTION
**Pull request by Hmeister-real**
## About The Pull Request

Also makes gutting people more consistent

## Why It's Good For The Game

insta killing by abusing the left over mortal wounds is stupid, fixed

### CODE REVIEW

File: code/datums/wounds/types/slashes.dm [hunk 1]
```diff
@@ -61,7 +61,7 @@
 	/// Organs we can disembowel associated with chance to disembowel
 	var/static/list/affected_organs = list(
 		ORGAN_SLOT_STOMACH = 100,
-		ORGAN_SLOT_LIVER = 50,
+		ORGAN_SLOT_LIVER = 100,
```
Comment (Cyprex) [inline code/datums/wounds/types/slashes.dm:64]: Why this change?

**Comment by ikrarkjr** [comment]
You have to crit them multiple times in a double accuracy spot, it's not even close to an instakill.

**Comment by Hmeister-real** [comment]
> You have to crit them multiple times in a double accuracy spot, it's not even close to an instakill.

Ears are the least protected place on the with almost every helmet, including the eyes and mouth. 2 crits for an instakill has no place here, it has been replaced by paralysation + brain damage which is effectively the same except you aren't fragging someone just disabling them.

File: code/datums/wounds/types/slashes.dm (hunk 1 above)
Comment (Hmeister-real) [reply code/datums/wounds/types/slashes.dm:64]: Consistency, now it's guaranteed to be 2 organs rather than a chance at 2

**Comment by kepesk1** [comment]
+1, ear insta kills are just exploit frags. i died twice just today from fencers  with rapiers used to double crit insta kill, not even an opportunity to RP with people who do this

**Comment by katsukai** [comment]
Make skeletons more immortal, coal take, coal change.

**Comment by john-pawson** [comment]
> Ears are the least protected place on the with almost every helmet

That is just not true, almost every single helmet has ear protection other than the basic iron skullcap helmet and the horned ones.
The basic sallet, the kettle helmet, all the coifs, all of the leather helmets, and all of the heavy helmets protect the ears, even the bervor, probably the most common neck piece, protects them.

**Comment by Hmeister-real** [comment]
> > Ears are the least protected place on the with almost every helmet
> 
> That is just not true, almost every single helmet has ear protection other than the basic iron skullcap helmet and the horned ones. The basic sallet, the kettle helmet, all the coifs, all of the leather helmets, and all of the heavy helmets protect the ears, even the bervor, probably the most common neck piece, protects them.

this is just, entirely wrong. Most helmets do not protect the ears, coifs don't protect ears and bervor absolutely doesn't protect the ears. this is purely just, made up bullshit lol

**Comment by john-pawson** [comment]
> > > Ears are the least protected place on the with almost every helmet
> > 
> > 
> > That is just not true, almost every single helmet has ear protection other than the basic iron skullcap helmet and the horned ones. The basic sallet, the kettle helmet, all the coifs, all of the leather helmets, and all of the heavy helmets protect the ears, even the bervor, probably the most common neck piece, protects them.
> 
> this is just, entirely wrong. Most helmets do not protect the ears, coifs don't protect ears and bervor absolutely doesn't protect the ears. this is purely just, made up bullshit lol

Here is, a screenshot, of every single helmet and armor that protects the ears. Including the bervor and coif.
Captura de pantalla 2024-10-02 003755
Captura de pantalla 2024-10-02 003810
(I thought the coif was obvious enough. But i guess not? I'm just surprised.)
Captura de pantalla 2024-10-02 003930
Captura de pantalla 2024-10-02 003940
Captura de pantalla 2024-10-02 003947
Captura de pantalla 2024-10-02 003957
Captura de pantalla 2024-10-02 004121
Captura de pantalla 2024-10-02 004130
Captura de pantalla 2024-10-02 004138
Captura de pantalla 2024-10-02 004154
Captura de pantalla 2024-10-02 004205
Captura de pantalla 2024-10-02 003911
Captura de pantalla 2024-10-02 003919

I won't put up every single heavy helmet, since they all cover the entire head, and therefore, the ears. Here's another screenshot if you need it as a reminder.
Captura de pantalla 2024-10-02 005223
And here's the define for FULL_HEAD if you need that too.
Captura de pantalla 2024-10-02 005532
I don't think i need to say much else. Please look into it before saying things like this.

**Comment by Hmeister-real** [comment]
> > > > Ears are the least protected place on the with almost every helmet
> > > 
> > > 
> > > That is just not true, almost every single helmet has ear protection other than the basic iron skullcap helmet and the horned ones. The basic sallet, the kettle helmet, all the coifs, all of the leather helmets, and all of the heavy helmets protect the ears, even the bervor, probably the most common neck piece, protects them.
> > 
> > 
> > this is just, entirely wrong. Most helmets do not protect the ears, coifs don't protect ears and bervor absolutely doesn't protect the ears. this is purely just, made up bullshit lol
> 
> Here is, a screenshot, of every single helmet and armor that protects the ears. Including the bervor and coif. Captura de pantalla 2024-10-02 003755 Captura de pantalla 2024-10-02 003810 (I thought the coif was obvious enough. But i guess not? I'm just surprised.) Captura de pantalla 2024-10-02 003930 Captura de pantalla 2024-10-02 003940 Captura de pantalla 2024-10-02 003947 Captura de pantalla 2024-10-02 003957 Captura de pantalla 2024-10-02 004121 Captura de pantalla 2024-10-02 004130 Captura de pantalla 2024-10-02 004138 Captura de pantalla 2024-10-02 004154 Captura de pantalla 2024-10-02 004205 Captura de pantalla 2024-10-02 003911 Captura de pantalla 2024-10-02 003919
> 
> I won't put up every single heavy helmet, since they all cover the entire head, and therefore, the ears. Here's another screenshot if you need it as a reminder. Captura de pantalla 2024-10-02 005223 And here's the define for FULL_HEAD if you need that too. Captura de pantalla 2024-10-02 005532 I don't think i need to say much else. Please look into it before saying things like this.

the leather helmet, the bandana, the tricorn and volfs helmet won't stop stabs at the ear for more than a few swings, only blunt attacks. also 4 of the helmets listed here are all gimmick patron helmets that are very rarely actually used by anyone, and also the chain coif won't stop stabs to the rear since it's meant to stop cuts and chops

**Comment by john-pawson** [comment]
> the leather helmet, the bandana, the tricorn and volfs helmet won't stop stabs at the ear for more than a few swings, only blunt attacks. 

They do stop attacks from the ears, just are easy to penetrate. And they protect the ear. But it does stop stabs since most common stabbing attacks like the rapier/sword don't have enough armor penetration to go through it.
For example, the sword stab has only an armor pen of 20 vs the helmet's value of 37, so it doesn't penetrate.
But the dagger's stab has an armor pen of 40, and it does.
But that's kind of the point of leather armors, they're supposed to be cheaper and worse than the metal versions, but it does protect them.

> Also 4 of the helmets listed here are all gimmick patron helmets that are very rarely actually used by anyone.

Well there's only 4 helmets that don't have ear protection and are very rarely actually used by anyone, but that didn't stop you from saying that the majority of helmets didn't have ear protection. 

Also: Unrelated. They're still used by clerics and templars usually and still are helmets that protect the ears. I showed you every helmet that protected the ears. And they're helmets that do.

> And also the chain coif won't stop stabs to the rear since it's meant to stop cuts and chops

It does. It has a low armor and it can be penetretated by high armor penetration attacks, but it does stop stabs since most stabbing attacks like the rapier/sword don't have enough armor penetration to go through it. Only special ones like the dagger's pick intent or the estoc's thrust.
For example: The chain coif has a stab protection of 45
Dagger's stab has an armor pen of 40, and it doesn't penetrate
But the estoc's stab while weilded has an armor pen of 60, and it does go through.

Not only THAT, but the chain coif also prevents specifically crits from stabs when the attack does go through too, including cuts and chops.
Captura de pantalla 2024-10-02 085550

**Comment by katsukai** [comment]
Please do not merge salt PRs.

**Comment by BlueMemesauce** [comment]
Remove mortal = FALSE, its false by default

**Comment by Red-Honk** [comment]
> > the leather helmet, the bandana, the tricorn and volfs helmet won't stop stabs at the ear for more than a few swings, only blunt attacks.
> 
> They do stop attacks from the ears, just are easy to penetrate. And they protect the ear. But it does stop stabs since most common stabbing attacks like the rapier/sword don't have enough armor penetration to go through it. For example, the sword stab has only an armor pen of 20 vs the helmet's value of 37, so it doesn't penetrate. But the dagger's stab has an armor pen of 40, and it does. But that's kind of the point of leather armors, they're supposed to be cheaper and worse than the metal versions, but it does protect them.
> 
> > Also 4 of the helmets listed here are all gimmick patron helmets that are very rarely actually used by anyone.
> 
> Well there's only 4 helmets that don't have ear protection and are very rarely actually used by anyone, but that didn't stop you from saying that the majority of helmets didn't have ear protection.
> 
> Also: Unrelated. They're still used by clerics and templars usually and still are helmets that protect the ears. I showed you every helmet that protected the ears. And they're helmets that do.
> 
> > And also the chain coif won't stop stabs to the rear since it's meant to stop cuts and chops
> 
> It does. It has a low armor and it can be penetretated by high armor penetration attacks, but it does stop stabs since most stabbing attacks like the rapier/sword don't have enough armor penetration to go through it. Only special ones like the dagger's pick intent or the estoc's thrust. For example: The chain coif has a stab protection of 45 Dagger's stab has an armor pen of 40, and it doesn't penetrate But the estoc's stab while weilded has an armor pen of 60, and it does go through.
> 
> Not only THAT, but the chain coif also prevents specifically crits from stabs when the attack does go through too, including cuts and chops. Captura de pantalla 2024-10-02 085550

Then explain the decapitations through unbroken chained coifs, as well as the delimbing of legs at chassues.

**Comment by john-pawson** [comment]
> Then explain the decapitations through unbroken chained coifs, as well as the delimbing of legs at chassues.

Oh that's actually a fun one! (Actively trying to fix it too btw) It's a bug on the system of armor that it usually checks for a single layer of armor, so for example if you're wearing a gambeson and chain leggings, and (for some reason, still looking into it) the gambeson is ABOVE the chain leggings, any attack that goes through the gambeson's armor simply just completely ignores the chausses armor, dealing the full damage and not preventing crits, this is the N°1 reason for bolts going through plate armor actually!
Feel free to test it out yourself by putting on ceirtain combinations of gambeson/chainmaille + Plate armor, punching yourself, and then seeing in amusement how the gambeson/chainmaille is the one taking the damage even though it's on shirt slot/under the plate armor.

**Comment by Cyprex** [comment]
Closed on Gyran's request

**Pull request closed without merging**